tests/seed/accounts/test_module.py

```
@@ -0,0 +1,99 @@
+class DummyAsyncIterator:
+    def __init__(self, modules):
+        self._iterator = iter(modules)
+
+    def __aiter__(self):
+        return self
+
+    async def __anext__(self):
+        try:
+            return next(self._iterator)
+        except StopIteration:
+            raise StopAsyncIteration
```

Ruff (0.14.7)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)